nengo/processes.py

```diff
@@ -371,31 +411,39 @@ def __init__(self, data, interpolation='zero', **kwargs):
         self.interpolation = interpolation
 
         super(Piecewise, self).__init__(
-            default_size_in=0, default_size_out=self.yp[0].size, **kwargs)
+            default_size_in=0, default_size_out=self._shape_out()[0], **kwargs)
+
+    def _interpolatable(self):
```

Comment:
I had initially thought of tracking this in the ``PiecewiseDataParam`` class, but didn't know if we could retract any info from the descriptor class.

Reply:
Ah you're right, it is difficult to access attributes of descriptor classes. Never mind!